app/search/controllers.py

```diff
@@ -24,8 +24,8 @@ def search_packages():
             - in: query
               name: q
               type: string
-              required: true
-              description: search query string e.g. q=query publisher=pub
+              required: false
```

Comment:
@rufuspollock I agree this is completely unrelated. That is why is also mentioned in commit message about this change. But it should not be here.